lib/tasks/update_ration_commodity_id.rake

```diff
@@ -0,0 +1,38 @@
+namespace :cats do
+    namespace :ration do
+        desc "update commodity id in ration items with commodity category id"
+        @ration_items= []
+       
+            task update_commodity_on_ration: :environment do
+                puts "Started updating commodity id ...."
+                @total = RationItem.all.count
+                @i = 0
+                RationItem.find_each do |ration_item|
```

Comment:
We can load commodity and commodity_category here rather than hitting the DB multiple times.
E.g RationItem.includes(commodity: :commodity_category).find_each do |ration_item|